webapp/channels/src/components/admin_console/index.ts

```diff
@@ -95,4 +94,4 @@ const connector = connect(mapStateToProps, mapDispatchToProps);
 
 export type PropsFromRedux = ConnectedProps<typeof connector>;
 
-export default withRouter(connector(AdminConsole));
+export default connector(AdminConsole);
```

Comment:
Why does this change here?

Reply:
The types were getting messed up, and after some testing, it looked like this was not needed here. The router information was already added by the parent component and after some quick test, I saw no problem.

Other similar components instantiated in the root file do not have the `withRouter`.